web/src/main/angular/src/app/core/components/transaction-id-search/transaction-id-search-container.component.ts

```diff
@@ -0,0 +1,53 @@
+import {
+    Component,
+    ChangeDetectionStrategy,
+    Output,
+    EventEmitter
+} from '@angular/core';
+
+import {
+    UrlRouteManagerService
+} from 'app/shared/services';
+import { UrlPath } from 'app/shared/models';
+
+@Component({
+    selector: 'pp-transaction-id-search-container',
+    templateUrl: './transaction-id-search-container.component.html',
+    styleUrls: ['./transaction-id-search-container.component.css'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class TransactionIdSearchContainerComponent {
+    @Output() outSearchId = new EventEmitter<{txId: string}>();
+
```

Comment:
There needs to be a property `searchUseEnter` if you take a look the template.
```suggestion
searchUseEnter = true;
```